docs/analyze.rst

```diff
@@ -49,23 +50,22 @@ single benchmark::
 Plotting of benchmark results
 -----------------------------
 
+  .. warning::
+
+   The function |mdbenchmark.analyze.plot|_ was deprecated with version 2.0. You should migrate to the newer ``mdbenchmark plot``, as it provides more functionality and the former version will be removed in the future.
+
 MDBenchmark provides a quick and simple way to plot the results of the
```

Comment:
But this still has the problem that multiple modules and gpu/cpu will not pbe handled correctly. We should make clear that the functionality is very limited and in a way "buggy"

Reply:
No, for this we have the old documentation that describes exactly the use case. Users should read the documentation for the version they are using.